examples/configmapgen_controller.rs

```diff
@@ -120,7 +113,11 @@ async fn main() -> Result<()> {
 
     Controller::new(cmgs, ListParams::default())
         .owns(cms, ListParams::default())
-        .run(reconcile, error_policy, Context::new(Data { client }))
+        .run(
+            reconcile,
+            || backoff::backoff::Constant::new(Duration::from_secs(1)),
```

Comment:
The lambda here is slightly awkward. Is it there because we need to reset it on success? If so, maybe we could cache a clone of it in the applier?

Reply:
Nevermind, I understand it now.